iron-overlay-behavior.html

```diff
@@ -405,7 +405,7 @@
         if (this.noCancelOnOutsideClick) {
           this._applyFocus();
         } else {
-          this.cancel();
+          this.cancel(event);
```

Comment:
Should the `event` be passed also on the method [`_onCaptureKeyDown`](https://github.com/tuct/iron-overlay-behavior/blob/master/iron-overlay-behavior.html#L418)?